### PR TITLE
UI-2240: Custom icon marker for px-map

### PR DESCRIPTION
**Pull request by sheela-svmx**
Support to show numbered markers and home markers on map 

To show the numbered pins on map like below, we need to extend px-map-marker-static and px-map-marker-symbol polymer components. Currently, those 2 marker components did not support to customize marker's icon 

Reference [Jira](https://servicemax.atlassian.net/browse/UI-2240)

### CODE REVIEW

**Comment by hanzge** [comment]
just remember to resolve conflict

File: px-custom-icon-marker.html [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+  <template>
+    <style>
+      :host {
+        display: none;
+      }
+
+    </style>
+    <slot></slot>
+  </template>
+  <script>
```
Comment (mallik-svmx) [inline px-custom-icon-marker.html]: Move the script block outside of Polymer <dom-module

**Comment by sheela-svmx** [comment]
@mallik-svmx and @joshsylvester addressed and tested

**Comment by sheela-svmx** [comment]
2 approvals merging

**Pull request merged**